arch/inst/C/c.srai.yaml

```diff
@@ -12,15 +12,22 @@ definedBy:
   anyOf:
     - C
     - Zca
-base: 64
 assembly: xd, shamt
 encoding:
-  match: 100-01--------01
-  variables:
-    - name: shamt
-      location: 12|6-2
-    - name: rd
-      location: 9-7
+  RV32:
+    match: 100001--------01
+    variables:
+      - name: shamt
+        location: 6-2
```

Comment:
I think both RV32 and RV64 stanzas, "shamt" variables need "not: 0".
The spec says, "C.SRAI is defined analogously to C.SRLI" and (for C.SRLI) "For RV32C and RV64C, the shift amount must be non-zero".